src/Microsoft.AspNetCore.ResponseCaching/ResponseCache/CachedResponse.cs

```diff
@@ -5,10 +5,12 @@
 
 namespace Microsoft.AspNetCore.ResponseCaching
 {
-    internal class ResponseCachingEntry
+    public class CachedResponse
     {
         public int StatusCode { get; set; }
+
         internal IHeaderDictionary Headers { get; set; } = new HeaderDictionary();
```

Comment:
No particular reason, could make everything internal until there's a reason to make them public. I don't see why these would be used by anyone else other than response caching middleware.